README.md

```diff
@@ -23,8 +23,8 @@ Vitess Operator Version | Recommended Vitess Versions | Recommended Kubernetes V
 `v2.4.*` | `v10.0.*`                   | `v1.15.*`, `v1.16.*`, or `v1.17.*`
 `v2.5.*` | `v12.0.*`                   | `v1.17.*`, `v1.18.*`, or `v1.19.*`
 `v2.6.*` | `v12.0.*`, or `v13.0.*`     | `v1.20.*`, `v1.21.*`, or `v1.22.*`
-`v2.7.*` | `v14.0.*`                   | `v1.20.*`, `v1.21.*`, or `v1.22.*`
-`latest` | `latest`                    | `v1.20.*`, `v1.21.*`, or `v1.22.*`
+`v2.7.*` | `v14.0.*`                   | `v1.20.*`, `v1.21.*`, `v1.22.*`, `v1.23.*`, `v1.24.*`, or `v1.25.*`
+`latest` | `latest`                    | `v1.20.*`, `v1.21.*`, `v1.22.*`, `v1.23.*`, `v1.24.*`, or `v1.25.*`
```

Comment:
I think this is incorrect, we should propably leave the `v2.7.*` row as it was. The next patch release of `v2.7` will support k8s `1.25.*` but the current and previous versions of `v2.7` do not. Perhaps we can add a `*` and detail the `*` right below the table to explain that only `v2.7.5` (the next patch release on this branch) will support k8s `1.25`.

Reply:
done 👍